addons/sale/models/sale.py

```diff
@@ -1283,3 +1283,16 @@ def _analytic_compute_delivered_quantity(self):
     def _is_delivery(self):
         self.ensure_one()
         return False
+
+
+# Try not to forward port this, and the whole commit in Master
+# Read the commit message
+class MailComposeMessage(models.TransientModel):
```

Comment:
Shouldn't this be here: https://github.com/odoo/odoo/blob/4f9fc188d398f45280ddbf9c61ecfad1ea476e92/addons/sale/wizard/mail_compose_message.py#L16